src/main/flight/pid.c

```diff
@@ -438,6 +438,92 @@ static float accelerationLimit(int axis, float currentPidSetpoint)
     return currentPidSetpoint;
 }
 
+static timeUs_t crashDetectedAtUs;
+
+void detectAndHandleCrashRecovery(
```

Comment:
Ok, for some reason the names of this and the next function are swapped now. This function only ever sets ` inCrashRecoveryMode = false`, so it should probably be `checkDisableCrashRecovery()`.

Reply:
Ok I think it should be something like:

handleCrashRecovery()

detectAndSetCrashRecovery()

Reply:
Sounds good to me.